pandas/core/indexes/base.py

```diff
@@ -1515,6 +1515,20 @@ def sortlevel(self, level=None, ascending=True, sort_remaining=None):
         -------
         Index
         """
+        if not isinstance(ascending, (list, bool)):
```

Comment:
these should be checked at a much lower level, where they are used; the reason these are fairly generic things not specific to Index in this case